notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -73,6 +73,27 @@
         "id": "k-82uiXlTjvw"
```

Comment:
<p>Don't use first-person. Should be <code>you validate the endpoint by making a simple prediction to it.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>